src/main/java/com/android/tools/build/bundletool/commands/BuildBundleCommand.java

```diff
@@ -252,6 +256,12 @@ public void execute() {
           AppBundle.buildFromModules(
               modulesWithTargeting.build(), bundleConfig, getBundleMetadata());
 
+      Path outputDirectory = getOutputPath().getParent();
```

Comment:
As far as I understood, `getOutputPath()` will always return a path to a .aab, right? So getting the parent in all cases should be fine?